cmd/convox/env.go

```diff
@@ -121,12 +129,49 @@ func cmdEnvGet(c *cli.Context) error {
 	return nil
 }
 
+// Validate environment variable keypair format; trim spaces and surrounding single quotes.
+func cleanEnvPair(value string) (string, error) {
+	// Don't let CLI swallow panics. (ref. urfave/cli/issues/451)
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Println("ERROR: ", r, string(debug.Stack()))
+		}
+	}()
+
+	parts := strings.SplitN(value, "=", 2)
+	if len(parts) != 2 {
+		return "", errors.New("Environment variables should be defined in key=value format. You specified: " + value)
+	}
+
+	if key := strings.TrimSpace(parts[0]); key != "" {
+		val := parts[1]
+		if len(val) == 0 {
+			return "", errors.New("Can't set " + key + " to an empty value; try `convox env unset`.")
```

Comment:
Here's the flow as I understand it:

`cmd/convox/env.go` in func `cmdEnvSet`:
`    _, releaseID, err := rackClient(c).SetEnvironment(app, strings.NewReader(data))`

`client/environment.go` in func `SetEnvironment`:
`    res, err := c.PostBodyResponse(fmt.Sprintf("/apps/%s/environment", app), body, &env)`

`api/controllers/routes.go`:
`router.HandleFunc("/apps/{app}/environment", api("environment.set", EnvironmentSet)).Methods("POST")`

`api/controllers/environment.go` in func `EnvironmentSet`:
`    releaseID, err := models.PutEnvironment(app, models.LoadEnvironment(body))`

When I put `fmt.Println(body)` or `log.Panic(body)` in the `EnvironmentSet` func, nothing gets printed when I run `convox env set`. Is that expected?